cypress/support/index.js

```diff
@@ -53,3 +53,11 @@ if (Cypress.env("security_enabled")) {
   Cypress.env("opensearch", `http://${Cypress.env("opensearch_url")}`);
   Cypress.env("opensearch_dashboards", `http://${Cypress.env("opensearch_dashboards_url")}`);
 }
+
+const resizeObserverLoopErrRe = /^[^(ResizeObserver loop limit exceeded)]/
+Cypress.on('uncaught:exception', (err) => {
```

Comment:
What's this for?

Reply:
My Policy Details test is throwing it, I believe between when the page appears and the API returns with the policy info. Researched and most places I found indicated that that error is benign, but it was causing the test to fail. I don't know what's causing it to appear on the Policy Details page, and not other similar details pages. This allows cypress to ignore that error and not fail the test because of it.

Reply:
The error message itself is concerning, seems like something going into an infinite loop. If it's not seen on any other page then perhaps we can rewrite to make it go away. Let's create a GitHub issue for it along w/ links you read.